package.json

```diff
@@ -43,7 +43,7 @@
     "downshift": "^3.1.12",
     "elasticlunr": "^0.9.5",
     "fs-extra": "3.0.1",
-    "mapbox-gl": "^0.48.0",
+    "mapbox-gl": "^1.4.1",
```

Comment:
you think this will have any affect on the other maps? hopefully not.

Reply:
i'll answer my own question here. local develop branch build, merged in this feature branch, npm install, and tested the maps. they seem good.

Reply:
I don't think it should cause any trouble. I checked the other maps and they seemed to function alright.